### DISCUSSION TITLE
Possibility for multiple images in a single recipe?

### DISCUSSION THREAD

**Discussion by tueboesen** [TandoorRecipes/recipes#3395] [General]
I'm pretty new to tandoor, so this is likely just a rookie mistake. But I can't seem to find any way of adding additional images to a recipe, specifically images at certain steps, such that people can see what the food is supposed to look like at that step. Does anyone know if this is possible and if so how?

**Comment by smilerz** [comment]
![image](https://github.com/user-attachments/assets/e69df267-89d2-47ef-b5bc-6e326932903c)

**Reply by tueboesen** [reply]
That was exactly what I didn't find, thank you!
